modules/light-clients/08-wasm/types/keys.go

```diff
@@ -9,4 +9,7 @@ const (
 
 	// Wasm is the client type for IBC light clients created using 08-wasm
 	Wasm = ModuleName
+
+	// KeyChecksums is the key under which all checksums are stored
+	KeyChecksums = "checksums"
```

Comment:
should we mark this as deprecated? or add a note that it was deleted in the upgrade to v8?

Reply:
we could put into the migrations file?

Reply:
either works, I think its fine to leave where it is if its marked as deprecated now :)

Reply:
I thought about that as well. Happy with either approach